sdcclient/_scanning.py

```diff
@@ -323,6 +307,49 @@ def get_pdf_report(self, image, tag=None, date=None):
 
         return [True, res.content]
 
+    def import_image(self, infile):
+        '''**Description**
+            Import an image archive
+
+        **Arguments**
+           - infile: An image archive file
+
+        **Success Return Value**
+            A JSON object representing the image that was imported.
+        '''
+        try:
+            m = MultipartEncoder(
+                fields={'archive_file': (infile, open(infile, 'rb'), 'text/plain')}
+            )
+            url = self.url+"/api/scanning/v1/import/images"
+
+            headers = {'Authorization': 'Bearer ' + self.token, 'Content-Type': m.content_type}
+            res = requests.post(url, data=m, headers=headers)
+            if not self._checkResponse(res):
+                return [False, self.lasterr]
+
+            return [True, res.json()]
+
+        except Exception as err:
+            print(err)
+
+    def get_anchore_users_account(self):
```

Comment:
How can this function be useful? Asking only because I have not much context about it (it may be useful to add more color in the description for other users as well).

Reply:
This API is needed to fetch the username for running inline-scan. The script uses this API here https://github.com/sysdiglabs/secure-inline-scan/blob/inline-scan/inline_scan.sh#L434